ONSdigital/ras-rm-secure-message-v2
# Add database structure

# What and why?

# How to test?

- Create a new schema in your dev database called `securemessages`
- Run the application (note that /deploy is currently broken; message me or Mark for a local command that does the same job)
- Ensure that it makes 2 tables in that schema: `message` (id, thread_id, body, sent_at, is_from_internal, sent_by) and `thread` (id, subject, category, is_closed, closed_by_id, closed_at, case_id, ru_ref, survey_id, assigned_internal_user_id, respondent_id, is_read_by_respondent, is_read_by_internal)
- This can't be tested by the business, so please ensure you dev test!

# Jira
RAS-776
